website/docs/r/metric_ruleset.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "signalfx"
+page_title: "SignalFx: signalfx_metric_ruleset"
+sidebar_current: "docs-signalfx-resource-metric-ruleset"
+description: |-
+Allows Terraform to create and manage SignalFx Metric Rulesets
```

Comment:
Should this be "Allows Terraform to create and manage Splunk Infrastructure Monitoring metric rulesets"? We don't refer to SignalFx in the docs any more